env0/data_project.go

```diff
@@ -162,14 +182,64 @@ func getProjectByName(name string, parentId string, parentName string, meta inte
 	return projectsByName[0], nil
 }
 
+func pathMatches(path, parentIds []string, meta interface{}) (bool, error) {
+	if len(path) != len(parentIds) {
```

Comment:
so if we have a1|a2|a3|searchedProject
and we put a1|a2 as the path we would not find it, is that intended?

Reply:
I tend to agree. We could support prefixes as well.
I don't know if that's what @away168 had in mind... but why not...

modifying to:
```
	if len(path) > len(parentIds) {
		return false, nil
	}
```